### PR TITLE
P18e AUV and Redmine #15120 fix

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP05MOAS-A6263/CP05MOAS-A6263_R00014_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.auv_eng.auv.auv_eng_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00014/*.txt,CP05MOAS-A6263-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.flort_kn.auv.flort_kn_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00014/*.txt,CP05MOAS-A6263-01-FLORTN000,recovered_host,Available,
+mi.dataset.driver.dosta_ln.auv.dosta_ln_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00014/*.txt,CP05MOAS-A6263-02-DOSTAN000,recovered_host,Available,
+mi.dataset.driver.ctdav_nbosi.auv.ctdav_nbosi_auv_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00014/*.txt,CP05MOAS-A6263-03-CTDAVN000,recovered_host,Available,
+mi.dataset.driver.nutnr_n.nutnr_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00014/DATA/*.SUN,CP05MOAS-A6263-04-NUTNRN000,recovered_host,Available,
+mi.dataset.driver.adcpa_n.adcpa_n_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00014/ADCP/*.ADC,CP05MOAS-A6263-05-ADCPAN000,recovered_host,Available,
+mi.dataset.driver.parad_n.auv.parad_n_auv_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-A6263/R00014/*.txt,CP05MOAS-A6263-06-PARADN000,recovered_host,Available,
```
Comment (rtravis-whoi) [inline CP05MOAS-A6263/CP05MOAS-A6263_R00014_ingest.csv:8]: looks good

File: GI03FLMA/GI03FLMA_R00005_ingest.csv [hunk 2]
```diff
@@ -10,7 +10,7 @@ mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/GI03FLMA
 mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/cg_data/MSIOC_sn*/STATUS/STA*.DAT,GI03FLMA-RIM01-00-SIOENG000,recovered_host,Available,
 mi.dataset.driver.adcps_jln.sio.adcps_jln_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/cg_data/MSIOC_sn*/ADCPS/Raw/ADCPS/*.DAT,GI03FLMA-RIM01-02-ADCPSL003,recovered_host,Available,
 mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/instruments/ADCPS_sn*/*.000,GI03FLMA-RIM01-02-ADCPSL003,recovered_inst,Available,
-#mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMA-RIM01-02-CTDMOG000,recovered_host,Not Expected,Wildcard refdes; files have incorrect format due to broken inductive loop; redmine #15120
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_ct_recovered_driver,/omc_data/whoi/OMC/GI03FLMA/R00005/cg_data/MSIOC_sn*/CTD/CTD*.DAT,GI03FLMA-RIM01-02-CTDMOG000,recovered_host,Available,Wildcard refdes
```
Comment (rtravis-whoi) [inline GI03FLMA/GI03FLMA_R00005_ingest.csv:13]: looks good - verified data available on raw data repo

Comment (snwhite) [reply GI03FLMA/GI03FLMA_R00005_ingest.csv:13]: Looks good.

File: CP05MOAS-A6263/CP05MOAS-A6263_R00014_ingest.csv (hunk 1 above)
Comment (snwhite) [reply CP05MOAS-A6263/CP05MOAS-A6263_R00014_ingest.csv:8]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**